c_library/gateway_c.cpp

```diff
@@ -109,3 +109,41 @@ const char* MuxyGateway_GetProductionURL(MuxyGateway Gateway)
 	gateway::SDK* SDK = static_cast<gateway::SDK*>(Gateway.SDK);
     return SDK->GetProductionURL().c_str();
 }
+
+void MuxyGateway_RunPoll(MuxyGateway Gateway, GW_PollConfiguration cfg)
```

Comment:
I'm curious about the difference in prefix `MuxyGateway_` vs `GW_`. Why have two?

Reply:
And above this is a third style: `Gateway_ForeachPayload`